src/Microsoft.ML.HalLearners/VectorWhitening.cs

```diff
@@ -35,52 +35,48 @@ namespace Microsoft.ML.Transforms.Projections
 {
     public enum WhiteningKind
     {
+        /// <summary>
+        /// PCA whitening.
+        /// </summary>
```

Comment:
Can you make sure that there are explanations and/or links for explanations for PCA and ZCA whitening in the `remarks` for this transform in the catalog?

Reply:
ZCA whitening ensures that the average covariance between whitened
and original variables is maximal.
In contrast, PCA  whitening lead to maximally compressed whitened variables, as
measured by squared covariance.


---
In reply to: [254010489](https://github.com/dotnet/machinelearning/pull/2366#discussion_r254010489) [](ancestors = 254010489)